src/shared/error.filter.ts

```diff
@@ -0,0 +1,27 @@
+import { ExceptionFilter, Catch, ArgumentsHost, Logger, HttpException, HttpStatus } from '@nestjs/common';
+
+@Catch()
+export class ErrorFilter implements ExceptionFilter {
+  catch(exception: HttpException, host: ArgumentsHost) {
+    const ctx = host.switchToHttp();
+    const response = ctx.getResponse();
+    const request = ctx.getRequest();
+    const status = exception.getStatus ? exception.getStatus() : HttpStatus.INTERNAL_SERVER_ERROR;
```

Comment:
looks like here you will have exceptions even without status codes. Or I have not understood it...
Where is it initialized?

Reply:
fixed